docs/admin/install.rst

```diff
@@ -389,9 +391,77 @@ This command will take a considerable amount of time and approximately 4GB of ba
 Test the Workstation
 ~~~~~~~~~~~~~~~~~~~~
 
-To start the SecureDrop Workstation client, double-click the SecureDrop desktop icon that was set up by the previous command. The preflight updater will start and check for updates. The system should be up-to-date and no updates should be required, but if updates are available follow the instructions in the preflight updater to apply them.
+To start the SecureDrop Client, double-click the SecureDrop desktop icon that was set up by the previous command. The preflight updater will start and check for updates. The system should be up-to-date and no updates should be required, but if updates are available follow the instructions in the preflight updater to apply them.
+
+Once the update check is complete, the SecureDrop Client will launch. Log in using an existing journalist account and verify that sources are listed and submissions can be downloaded, decrypted, and viewed.
+
+Post-Install Configuration
+--------------------------
+Once you have verified that you can log into the SecureDrop Client, you may want to consider making additional configuration changes:
+
+- setting up password management
```

Comment:
A workstation USB is attached during the installation phase, it would make sense to copy the keepassx db across at that point (as an optional step).
- this avoids repeatedly connecting said USB during the course of the install,
- it avoids fun with USB auto-attach and associated potential issues (thinking in particular of the one where encrypted volumes don't seem to get removed properly and create multiple `/dev/xvdiN` devices.),
- the docs don't have to explain the `vault` VM twice.

Reply:
I like that suggestion a lot especially because of the second point (auto-attach confusion). In organizations where journalist and admin are not the same person, it does mean a third USB drive (Journalist Workstation USB) may hold those credentials, but that's all the more reason to add it to the section upfront where users have to inventory USBs. Will rejig.

Reply:
This is now done in dd9242951e399f860eae68bf2ed42b75cf0d11ad and I feel it definitely streamlines the process, but please take a close look at those revised instructions as this is a big change and new issues may have crept in.